keyboards/ergodash/ergodash.h

```diff
@@ -5,23 +5,40 @@
 
 #ifdef KEYBOARD_ergodash_rev1
     #include "rev1.h"
+    // Used to create a keymap using only KC_ prefixed keys
+    #define LAYOUT_kc( \
+        L00, L01, L02, L03, L04, L05, L06, R00, R01, R02, R03, R04, R05, R06, \
+        L10, L11, L12, L13, L14, L15, L16, R10, R11, R12, R13, R14, R15, R16, \
+        L20, L21, L22, L23, L24, L25, L26, R20, R21, R22, R23, R24, R25, R26, \
+        L30, L31, L32, L33, L34, L35, L36, R30, R31, R32, R33, R34, R35, R36, \
+        LT0, LT1, LT2, LT3,      LT4, LT5, RT0, RT1,      RT2, RT3, RT4, RT5  \
+        ) \
+        LAYOUT( \
+            KC_##L00, KC_##L01, KC_##L02, KC_##L03, KC_##L04, KC_##L05, KC_##L06, KC_##R00, KC_##R01, KC_##R02, KC_##R03, KC_##R04, KC_##R05, KC_##R06, \
+            KC_##L10, KC_##L11, KC_##L12, KC_##L13, KC_##L14, KC_##L15, KC_##L16, KC_##R10, KC_##R11, KC_##R12, KC_##R13, KC_##R14, KC_##R15, KC_##R16, \
+            KC_##L20, KC_##L21, KC_##L22, KC_##L23, KC_##L24, KC_##L25, KC_##L26, KC_##R20, KC_##R21, KC_##R22, KC_##R23, KC_##R24, KC_##R25, KC_##R26, \
+            KC_##L30, KC_##L31, KC_##L32, KC_##L33, KC_##L34, KC_##L35, KC_##L36, KC_##R30, KC_##R31, KC_##R32, KC_##R33, KC_##R34, KC_##R35, KC_##R36, \
+            KC_##LT0, KC_##LT1, KC_##LT2, KC_##LT3,           KC_##LT4, KC_##LT5, KC_##RT0, KC_##RT1,           KC_##RT2, KC_##RT3, KC_##RT4, KC_##RT5  \
+        )
 #endif
 
-
-// Used to create a keymap using only KC_ prefixed keys
-#define LAYOUT_kc( \
-    L00, L01, L02, L03, L04, L05, L06, R00, R01, R02, R03, R04, R05, R06, \
-    L10, L11, L12, L13, L14, L15, L16, R10, R11, R12, R13, R14, R15, R16, \
-    L20, L21, L22, L23, L24, L25, L26, R20, R21, R22, R23, R24, R25, R26, \
-    L30, L31, L32, L33, L34, L35, L36, R30, R31, R32, R33, R34, R35, R36, \
-    LT0, LT1, LT2, LT3,      LT4, LT5, RT0, RT1,      RT2, RT3, RT4, RT5  \
-    ) \
-    LAYOUT( \
-        KC_##L00, KC_##L01, KC_##L02, KC_##L03, KC_##L04, KC_##L05, KC_##L06, KC_##R00, KC_##R01, KC_##R02, KC_##R03, KC_##R04, KC_##R05, KC_##R06, \
-        KC_##L10, KC_##L11, KC_##L12, KC_##L13, KC_##L14, KC_##L15, KC_##L16, KC_##R10, KC_##R11, KC_##R12, KC_##R13, KC_##R14, KC_##R15, KC_##R16, \
-        KC_##L20, KC_##L21, KC_##L22, KC_##L23, KC_##L24, KC_##L25, KC_##L26, KC_##R20, KC_##R21, KC_##R22, KC_##R23, KC_##R24, KC_##R25, KC_##R26, \
-        KC_##L30, KC_##L31, KC_##L32, KC_##L33, KC_##L34, KC_##L35, KC_##L36, KC_##R30, KC_##R31, KC_##R32, KC_##R33, KC_##R34, KC_##R35, KC_##R36, \
-        KC_##LT0, KC_##LT1, KC_##LT2, KC_##LT3,           KC_##LT4, KC_##LT5, KC_##RT0, KC_##RT1,           KC_##RT2, KC_##RT3, KC_##RT4, KC_##RT5  \
-    )
+#ifdef KEYBOARD_ergodash_rev2
+    #include "rev2.h"
```

Comment:
If each revision has a different `LAYOUT_kc` for each revision, it may be better to put these into the revision folders, along with the normal keymap.  But that's your call, since it doesn't make much of a difference.  It just "looks cleaner", IMO.